### DISCUSSION TITLE
Multiple choice users

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#348] [4. Bug Reports] [answered]
Hola!

Quería saber si en un survey cuando se elige un multiple choice de usuarios, debería aparecer uno mismo ?

El contexto de mi situación es que cree un creador de task en donde se define al asignado de la tarea, entonces tengo el campo obligatorio, pero si me la quiero asignar a mí msimo, no aparezco para seleccionarme. 

La buena practica seria dejar el campo opcional, y la rutina al intentar llenar el campo de asignado se encontraría con un campo vacio (?)

gracias!

**Comment by agallardol** [comment, marked as answer]
@ValeMP Estamos al tanto de este bug e intentaremos solucionarlo en los próximos releases

**Reply by ValeMP** [reply]
perfect gracias!

**Comment by ValeMP** [comment]
En todo caso, voy a dejar aquí como lo resolví.
1. Dejar el campo de asignado opcional.
2. en el bot para editar la tarea en el campo asignee deje la siguiente lógica: 
```$OUTPUT#get_answer#data|data|answers|0|data|[find=>identifier=ms_asignee]|process|[add=>string=($OUTPUT#get_answer#data|data|answers|0|user)]|0```
En donde concateno la respuesta con el usario que respondió el formulario, y luego seleccione el id d la primera posicion del arreglo. Así si hay uan repuesta, tomo el id de la respuesta. Sino tomo el id de la persona que respondió.
